README.md

```diff
@@ -92,6 +92,21 @@ pipeline {
 }
 ```
 
+### Detecting whether a build was started by the trigger in a script:
+```groovy
+def triggerCause = currentBuild.rawBuild.getCause(
+    org.jenkinsci.plugins.pipeline.github.trigger.IssueCommentCause
+)
+
+if (triggerCause.asBoolean()) {
```

Comment:
I think this can just be `if (triggerCause) {` assuming that `currentBuild.rawBuild.getCause()` returns `null` if no matching cause was found.

Reply:
[It](http://javadoc.jenkins-ci.org/hudson/model/Run.html#getCause-java.lang.Class-) has a [`@CheckForNull` annotation](http://findbugs.sourceforge.net/manual/annotations.html) and I remember seeing it as `null` for builds not triggered by a comment, so I think the assumption holds.

I used `asBoolean()`, because I thought that maybe more explicit is better. I'll change regardless.